specification/purview/resource-manager/Microsoft.Purview/preview/2024-04-01-preview/purview.json

```diff
@@ -1631,6 +1630,64 @@
         }
       }
     },
+    "AccountMergeInfo": {
+      "description": "The public Account Merge Info model.",
+      "type": "object",
+      "properties": {
+        "accountLocation": {
+          "description": "The account location of the *other* account in the merge operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "accountName": {
+          "description": "The account name of the *other* account in the merge operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "accountResourceGroupName": {
+          "description": "The resource group name of the *other* account in the merge operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "accountSubscriptionId": {
+          "description": "The subscription id of the *other* account in the merge operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "deprovisioned": {
+          "description": "The deprovisioned status of the account.\r\nOnly applicable for the secondary account.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "mergeStatus": {
+          "description": "The status of the merge operation.",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "InProgress"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "MergeStatus",
+            "modelAsString": true
+          }
+        },
+        "typeOfAccount": {
+          "description": "The type of account this account is for the merge operation.",
```

Comment:
Suggested rewording: 'This account's type, for the merge operation.'

Reply:
Updated the wording.